shell/app/node_main.cc

```diff
@@ -153,6 +153,7 @@ int NodeMain(int argc, char* argv[]) {
 
     exit_code = node::EmitExit(env);
     env->set_can_call_into_js(false);
```

Comment:
Does this line need to be removed?

Reply:
it's redundant, but it doesn't hurt to remove it.